readme.md

```diff
@@ -310,6 +310,7 @@ This section details services that have integrated Lighthouse data. If you're wo
 ## Related Projects
 Other awesome open source projects that use Lighthouse.
 
+* **[auto-lighthouse](https://github.com/TGiles/auto-lighthouse)** - a CLI for crawling a domain, generating mobile and desktop reports, and viewing them automatically after generation.
```

Comment:
might want to add this great description to the repo itself @TGiles this sounds awesome, and I'm not really sure what "automated Lighthouse auditing" meant on its own :)

Reply:
```suggestion
* **[auto-lighthouse](https://github.com/TGiles/auto-lighthouse)** - a CLI for crawling a domain and generating mobile and desktop reports for each page.
```

minor suggestion since the key features are crawling and mobile+desktop